tests/unittests/OperatorTest.cpp

```diff
@@ -928,21 +928,35 @@ TEST_P(InterpAndCPU, ScatterAssignQuantized) {
 }
 
 TEST_P(Operator, QuantizeAndDequantize) {
-  Tensor inputs(ElemKind::FloatTy, {1, 4});
-  inputs.getHandle() = {1, 1.2f, 0.5f, 1.3f};
+  Tensor input1(ElemKind::FloatTy, {1, 4});
+  Tensor input2(ElemKind::FloatTy, {1, 4});
+  input1.getHandle() = {1, 1.2f, 0.5f, 1.3f};
+  input2.getHandle() = {1.9f, 5.2f, 3.5f, 6.3f};
 
   auto *A = mod_.createVariable(ElemKind::FloatTy, {1, 4}, "A",
                                 VisibilityKind::Public);
+  auto *B = mod_.createVariable(ElemKind::FloatTy, {1, 4}, "B",
+                                VisibilityKind::Public);
 
   auto qType = mod_.uniqueType(ElemKind::Int8QTy, {1, 4}, 0.05, -138);
```

Comment:
make sure that values from the fp32 variables can be represented by {s,o} = {0.05, -138}

Reply:
changed to the same values as in input2, so i can ensure they can be dequantzied

Reply:
my point is that {0.05, -138} covers floating point range [0.05*(-128 + 138), 0.05 *(127+138)].
I was just asking to make sure that the values from input1 and input2 are within this range :).
I think it's better to have different values for input1 and input2 though.